src/pagination/pagination.spec.ts

```diff
@@ -0,0 +1,210 @@
+import {
+  iit,
+  it,
+  ddescribe,
+  describe,
+  expect,
+  inject,
+  injectAsync,
+  TestComponentBuilder,
+  beforeEach,
+  beforeEachProviders
+} from 'angular2/testing';
+
+import {Component} from 'angular2/angular2';
+
+import {NgbPagination} from './pagination';
+
+function getLink(nativeEl, idx) {
+  return nativeEl.querySelectorAll('li')[idx].querySelector('a');
```

Comment:
Hmm, looks legit, but doesn't work in practice (tests start to fail, at least in Chrome). Didn't dig into it too much.